src/compiler/diagnosticMessages.json

```diff
@@ -3408,6 +3408,22 @@
         "category": "Error",
         "code": 4110
     },
+    "This member cannot have an 'override' modifier because its containing class '{0}' does not extend another class.": {
+        "category": "Error",
+        "code": 4111
+    },
+    "This member cannot have an 'override' modifier because it is not declared in the base class '{0}'.": {
+        "category": "Error",
+        "code": 4112
+    },
+    "This member must have an 'override' modifier because it overrides a member in the base class '{0}'.": {
+        "category": "Error",
+        "code": 4113
+    },
+    "This parameter must convert into property declaration because it overrides a member in the base class '{0}'.": {

```

Comment:
```suggestion
This parameter property must be rewritten as a property declaration with an 'override' modifier because it overrides a member in base class '{0}'.
```